src/platform/packages/shared/kbn-alerts-ui-shared/src/alert_filter_controls/filter_group.test.tsx

```diff
@@ -589,8 +589,14 @@ describe(' Filter Group Component ', () => {
         />
       );
 
-      expect(consoleErrorSpy.mock.calls.length).toBe(1);
-      expect(String(consoleErrorSpy.mock.calls[0][0])).toMatch(URL_PARAM_ARRAY_EXCEPTION_MSG);
+      const componentErrors = consoleErrorSpy.mock.calls.filter(
+        (c) =>
+          !c[0]?.startsWith?.(
+            'Warning: ReactDOM.render'
```

Comment:
Fixing [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/266993#0194655f-1466-4504-adc9-6a30f42ad0ea) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/266993/jobs/0194655f-1466-4504-adc9-6a30f42ad0ea/artifacts/01946582-775f-4b81-b36b-e792746cf7fc) Jest Tests #3 / Filter Group Component Filter Changed Banner should ignore url params if there is an error in using them


The problem is that when running React@18 in legacy mode there is an additional log message that warns about running in legacy mode. Since this is the only place that got broken because of it, I haven't come up with anything better then just adjusting in place :)